ros2bag/test/resources/incomplete_qos_duration.yaml

```diff
@@ -4,4 +4,4 @@
   reliability: reliable
   durability: volatile
   deadline:
-    sec: 2147483647  # LONG_MAX
+    sec: 0
```

Comment:
I guess there's no real constants in yaml, but would it be worth to keep a comment in here?

Reply:
I've added comments to the human readable places where it is a special value. For this specific file, I changed it to `2` as some arbitrary choice, since this is an error test anyways